src/main/resources/application.properties

```diff
@@ -5,5 +5,5 @@ spring.datasource.password=kodilla_password
 spring.jpa.database=mysql
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy
+spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl

```

Comment:
skąd ta zmiana?

Reply:
IDE wywala mi błąd przy tym, dopiero po zmianie przechodzi. Kwestia uzgodnienia czy i jakiej strategii nazewnictwa będziemy używać?

Reply:
U mnie jest okej, może napisz jaki to błąd i uda się go rozwiązać? :)